app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -184,7 +184,7 @@ export const simulateEstimateGas = async (ethereumClientService: EthereumClientS
 		} as const
 	}
 	if (lastResult.status === 'failure') {
-		console.log(lastResult)
+		console.warn(`Gas estimate failure result: ${lastResult}`)
```

Comment:
I don't think this should be a waning, as its not really warning. Application just asked for gas estimate and it couldn't be estimated